examples/lightclient/lightclient.c

```diff
@@ -57,6 +57,7 @@
 
 #include "liblwm2m.h"
 #include "connection.h"
+#include "commandline.h"
```

Comment:
I put `validate_coap_block_size` in `commandline.c` because it is shared code for the examples. Therefore this include is need in the lightclient. Another place would be `core/utils.c` and `liblwm2m.h`.